library/core/src/num/mod.rs

```diff
@@ -773,6 +773,10 @@ macro_rules! from_str_radix_int_impl {
         #[stable(feature = "rust1", since = "1.0.0")]
         impl FromStr for $t {
             type Err = ParseIntError;
+            #[doc(alias = "atoi")]
+            #[doc(alias = "atol")]
+            #[doc(alias = "strtod")]
+            #[doc(alias = "strtol")]
```

Comment:
I figured these were all "close enough" since `int` and `long` don't have a fixed size.